lib/list.js

```diff
@@ -49,18 +50,20 @@ function validate (opts) {
 
 function defineCollectionUrl (opts) {
   const { collection, category } = opts;
-  const path = c.collectionPaths[collection];
-
+  const baseUrl = `${BASE_URL}/store/apps`;
+  const path = collection.startsWith('NEW_') ? `${baseUrl}/new` : `${baseUrl}/top`;
   return (category)
     ? `${path}/category/${category}`
     : path;
 }
 
-function buildUrl (opts) {
+function buildInitialUrl (opts) {
   const url = defineCollectionUrl(opts);
 
+  // For the initial collection lookup we force the language to english
+  // since we depend on collection names to identify the proper cluster later on
   const queryString = {
-    hl: opts.lang,
+    hl: 'us',
```

Comment:
This wouldn't give us wrong collections depending on the language that is being sent?

Reply:
I added a test for that case, the cluster is selected properly in English and then the list is parsed in whatever language you requested